src/Content/Backend/Solution/.template.config/template.json

```
@@ -226,22 +239,27 @@
 						"Monaco.Template.Backend.Application.Tests/Services/FileServiceTests.cs",
 						"Monaco.Template.Backend.Application.Tests/Features/File/**/*",
 						"Monaco.Template.Backend.Application.Tests/Features/Product/**/*",
-						"Monaco.Template.Backend.Common.Tests/Factories/Entities/DocumentFactory.cs",
-						"Monaco.Template.Backend.Common.Tests/Factories/Entities/ImageFactory.cs",
-						"Monaco.Template.Backend.Common.Tests/Factories/Entities/ProductFactory.cs",
+						"Monaco.Template.Backend.Domain.Tests/Factories/Entities/DocumentFactory.cs Monaco.Template.Backend.Common.Tests/Factories/Entities/DocumentFactory.cs",
```

I think this exclusion should only have 1 file path?

Also I think we are missing the exclusion for FileFactory.cs still.

```suggestion
						"Monaco.Template.Backend.Domain.Tests/Factories/Entities/DocumentFactory.cs",
						"Monaco.Template.Backend.Domain.Tests/Factories/Entities/FileFactory.cs",
```

---

Oops, my bad!

---

Done.

---

I've added the missing exclusion for FileFactory.